ui/src/app/kvm/components/KVMResourcesCard/KVMResourcesCard.tsx

```diff
@@ -22,37 +22,37 @@ const KVMResourcesCard = ({ id }: Props): JSX.Element => {
   );
 
   if (pod) {
-    const { cpu_over_commit_ratio, memory_over_commit_ratio, resources } = pod;
-    const { interfaces, memory } = resources;
-    const cores = resourceWithOverCommit(
-      resources.cores,
-      cpu_over_commit_ratio
-    );
-    const general = resourceWithOverCommit(
-      memory.general,
+    const {
+      cpu_over_commit_ratio,
+      memory_over_commit_ratio,
+      resources: {
+        cores,
+        interfaces,
+        memory: { general, hugepages },
+      },
+    } = pod;
+    const coresWithOver = resourceWithOverCommit(cores, cpu_over_commit_ratio);
+    const generalWithOver = resourceWithOverCommit(
+      general,
       memory_over_commit_ratio
     );
-    const hugepages = memory.hugepages; // Hugepages do not take over-commit into account
     return (
       <>
         <div className="kvm-resources-card">
           <RamResources
             dynamicLayout
-            general={{
-              allocated: general.allocated_tracked,
-              free: general.free,
-            }}
-            hugepages={{
-              allocated: hugepages.allocated_tracked,
-              free: hugepages.free,
-            }}
+            generalAllocated={generalWithOver.allocated_tracked}
+            generalFree={generalWithOver.free}
+            generalOther={generalWithOver.allocated_other}
+            hugepagesAllocated={hugepages.allocated_tracked}
+            hugepagesFree={hugepages.free}
+            hugepagesOther={hugepages.allocated_other}
           />
```

Comment:
I made the props flat for `RamResources` in order to be able to provide sensible defaults in the component, e.g. in the NUMA cards there can never be an option to provide `other` data since it's only concerned with the actual physical bit of hardware being used.